<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>acadamy_dialog</class>
 <widget class="QDialog" name="acadamy_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1056</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Acadamy</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_new_template">
       <property name="text">
        <string>New template</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_all_templates">
       <property name="text">
        <string>Templates</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_filter">
       <item>
        <widget class="QLabel" name="label_filter">
         <property name="text">
          <string>Filter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_filter"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget_all_templates"/>
     </item>
     <item>
      <widget class="QLabel" name="label_template_details">
       <property name="text">
        <string>Selected Template</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget_template_detail"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_back">
       <property name="text">
        <string>&lt;- Back (Esc)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="verticalStackedWidget_forms">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="page_splash">
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>500</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>../src/ui/mascot.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="margin">
          <number>50</number>
         </property>
         <property name="indent">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_new_template">
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_template_name"/>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Stat block areas:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_type_templates_page">
         <item>
          <property name="text">
           <string>Lore</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ability Scores</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Attributes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Items</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Actions</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_add_type_template">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Stack a template on top of this one</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_filter_templates_page">
           <property name="text">
            <string>Filter</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_filter_templates_page"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_templates_page"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_new_sub_template">
         <property name="text">
          <string>Stack</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_template_delete">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_lore">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_name_lore">
         <property name="text">
          <string>Lore Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_name_lore"/>
       </item>
       <item>
        <widget class="QLabel" name="label_description_lore">
         <property name="text">
          <string>Lore description</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_content_lore"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_delete_lore">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_attribute_score">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_attribute_score_name">
         <property name="text">
          <string>Attribute_score template name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_attribute_score_name"/>
       </item>
       <item>
        <widget class="QLabel" name="label_attribute_score">
         <property name="text">
          <string>score template</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_score_button_group">
         <item>
          <widget class="QComboBox" name="comboBox_attribute_score">
           <item>
            <property name="text">
             <string>Strength</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Dexterity</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Constitution</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Wisdom</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Intelligence</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Charisma</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_attribute_score">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>can have + or - to add or remove points from the base score. Or without them the value is replaced</string>
           </property>
           <property name="maxLength">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_attribute_score_new">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_attribute_score_delete">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_attribute">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_attribute_name">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_attribute_name"/>
       </item>
       <item>
        <widget class="QLabel" name="label_attribute_description">
         <property name="text">
          <string>description (optional)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_attribute_description"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_attribute_delete">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_item">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label_item_name">
         <property name="text">
          <string>Item Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_item_name"/>
       </item>
       <item>
        <widget class="QLabel" name="label_item_weight">
         <property name="text">
          <string>Item Weight</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_item_weight"/>
       </item>
       <item>
        <widget class="QLabel" name="label__item_description">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;Description: (Allows %d,%e) &lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_item_description"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_item_roll_table">
         <property name="text">
          <string>Add roll table</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_item_random_table">
         <property name="text">
          <string>Add random table</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_item_delete">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_action">
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="label_action_name">
         <property name="text">
          <string>Action name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_action_name"/>
       </item>
       <item>
        <widget class="QLabel" name="label_action_description">
         <property name="text">
          <string>Description: %d, %e for auto damage and elemental damage</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_action_description"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_action_delete">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_dice_roll">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_dice_roll_name">
         <property name="text">
          <string>Roll table name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_dice_roll_name"/>
       </item>
       <item>
        <widget class="QLabel" name="label_dice_roll_parent">
         <property name="text">
          <string>dice roll</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_parent_dice_roll">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_dice_roll_range">
         <property name="text">
          <string>Range: written 5-10 or 20 subtituting your own numbers of coarse.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_dice_roll_range"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_dice_roll_delete">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_random">
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QLabel" name="label_random_name">
         <property name="text">
          <string>Random table name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_random_name">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_random_delete">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
